### PR TITLE
Snapsdocs

### PR DESCRIPTION
**Pull request by mapachurro**
This is a draft PR for the initial MM Snaps documentation as per @rekmarks

### CODE REVIEW

File: docs/guide/common-terms.md [hunk 1]
```diff
@@ -2,296 +2,10 @@
 
 ## Words are Hard
 
-<p id="agoctitecs_p">
-  This is a list of terms you might encounter when using the MetaMask interface.
-</p>
+Every field has its terminology, and every specialization has its acronyms. You can't escape it.
 
-#### [You can also read in Spanish](https://github.com/faraggi/words-are-hard-es/blob/master/words-are-hard-es.md), thanks to [faraggi](https://github.com/faraggi)
+The decentralized web is certainly no exception, and if you're new to Web3, take a gander at MetaMask's [introduction to blockchain article](https://metamask.zendesk.com/hc/en-us/articles/360015489611) in our knowledgebase.
```
Comment (georgewrmarshall) [inline docs/guide/common-terms.md:7]: Should there be space between knowledge base?
```suggestion
The decentralized web is certainly no exception, and if you're new to Web3, take a gander at MetaMask's [introduction to blockchain article](https://metamask.zendesk.com/hc/en-us/articles/360015489611) in our knowledge base.
```

File: docs/guide/initializing-dapps.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+No matter which network you deploy your final dapp on, your user will need to be able to access it. As of v3.3.0, MetaMask makes available [wallet_switchEthereumChain](https://docs.metamask.io/guide/rpc-api.html#other-rpc-methods), allowing you to prompt the user to add a chain that you suggest, and switch to it using a confirmation dialogue.
 
 ## The Contract Address
 
-Every account in Ethereum has an address, whether it's an external key-pair account, or a smart contract. In order for any smart contract library to communicate with your contracts, they'll need to know its exact address.
+Every account in Ethereum has an address, whether it's an external key-pair account, or a smart contract. In order for any smart contract library to communicate with your contracts, they'll need to know its exact address. If you deployed a contract, you probably know how to find the address, but if you're just getting started, check out our Knowledgebase [explainer on finding contract addresses.](https://metamask.zendesk.com/hc/en-us/articles/360059683451-How-to-view-or-add-custom-token-contract-address)
```
Comment (georgewrmarshall) [inline docs/guide/initializing-dapps.md:13]: Should knowledge base be 2 words? [reference](https://www.knowledgeowl.com/home/knowledge-base-or-knowledgebase#:~:text=It%20is%20most%20commonly%20spelled,you%20wouldn't%20be%20alone.)
```suggestion
Every account in Ethereum has an address, whether it's an external key-pair account, or a smart contract. In order for any smart contract library to communicate with your contracts, they'll need to know its exact address. If you deployed a contract, you probably know how to find the address, but if you're just getting started, check out our knowledge base [explainer on finding contract addresses.](https://metamask.zendesk.com/hc/en-us/articles/360059683451-How-to-view-or-add-custom-token-contract-address)
```

**Review by georgewrmarshall — COMMENTED** [review]
Looks good! Just one question about knowledgebase or knowledge base :)

**Comment by rekmarks** [comment]
Superseded by #330.

**Pull request closed without merging**